pyproject.toml

```diff
@@ -62,6 +63,13 @@ markers = "platform_python_implementation == 'CPython'"
 
 [tool.pytest.ini_options]
 markers = "slow: Mark the test as slow. Can be skipped with --skip_slow"
+addopts = "--cov=manim --cov-report xml"
```

Comment:
I would like if the coverage printed on the CI logs, so remove `--cov-report xml` and in a later step run `coverage xml` command so that it can be uploaded to codecov.

Reply:
I think I can specify both outputs. Let's try it...

Reply:
It works (9d50c28)